SortingComparison.py

```diff
@@ -0,0 +1,89 @@
+from random import randint
+from timeit import timeit
+
+
+def data(start,stop,size):
+    lista = []
+    for i in range(int(size)):
+        nl = randint(start, stop)
+        lista.append(nl)
+    return lista
+
+
+def bubblesort(data):
+    nd = data[:]
```

Comment:
nd znowu- co to za twór? szkoda Ci miejsca na nazwy zmiennych? newData? no nie mam pojęcia